src/pages/evm/staking/WithdrawTab.vue

```diff
@@ -0,0 +1,170 @@
+<script setup lang="ts">
+import { computed, onBeforeMount, ref } from 'vue';
+import { useI18n } from 'vue-i18n';
+import { ethers } from 'ethers';
+
+import {
+    CURRENT_CONTEXT,
+    getAntelope,
+    useAccountStore,
+    useBalancesStore,
+    useChainStore,
+    useFeedbackStore,
+    useRexStore,
+    useUserStore,
+} from 'src/antelope';
+
+import WithdrawRaw from 'src/pages/evm/staking/WithdrawRaw.vue';
+import EVMChainSettings from 'src/antelope/chains/EVMChainSettings';
+import { formatWei } from 'src/antelope/stores/utils';
+
+
+const { t: $t } = useI18n();
+const uiDecimals = 2;
+const ant = getAntelope();
+const chainStore = useChainStore();
+const chainSettings = chainStore.currentChain.settings as EVMChainSettings;
+const userStore = useUserStore();
+const balanceStore = useBalancesStore();
+const accountStore = useAccountStore();
+const rexStore = useRexStore();
+const feed = useFeedbackStore();
+
+const systemToken = chainSettings.getSystemToken();
+const systemTokenSymbol = systemToken.symbol;
+const systemTokenDecimals = systemToken.decimals;
+const stakedToken = chainSettings.getStakedSystemToken();
+const stakedTokenSymbol = stakedToken.symbol;
+const stakedTokenDecimals = stakedToken.decimals;
+
+const loading = computed(() => feed.isLoading('withdrawEVMSystemTokens'));
+const allWithdrawals = computed(() => rexStore.getEvmRexData(CURRENT_CONTEXT)?.deposits ?? []);
+
+// prettyPrintToken(unstakingBalanceBn.value, systemToken.symbol)
+const withdrawableBalanceBn = computed(() => useRexStore().getRexData(CURRENT_CONTEXT)?.withdrawable ?? ethers.constants.Zero);
+
+// enable only if withdrawableBalanceBn > 0
+const withdrawEnabled = computed(() => withdrawableBalanceBn.value?.gt(0) ?? false);
```

Comment:
should withdrawEnabled also be false if loading is true?

Reply:
correct !!
fixed